vars.tf

```diff
@@ -8,6 +8,11 @@ variable "plan" {
   description = "Plan for instance"
 }
 
+variable "nodes" {
+  type        = number
+  description = "Number of instances for plan. *Deprecated* see https://registry.terraform.io/providers/cloudamqp/cloudamqp/latest/docs/resources/instance#nodes"
```

Comment:
Should we have a default null-value perhaps? I think for a "new" plan this would cause deprecation-warnings.